lib/modules/platform/types.ts

```diff
@@ -23,6 +23,7 @@ export interface PlatformResult {
   renovateUsername?: string;
   token?: string;
   gitAuthor?: string;
+  hostRules?: [HostRule]; // return these only if _additional_ rules/hosts are required
```

Comment:
```suggestion
  hostRules?: HostRule[]; // return these only if _additional_ rules/hosts are required
```
isn't that the right syntax? 🤔

the other one only allows a single array element of hostrule type. 🤔